content/en-US/api/fontStretch.md

```diff
@@ -0,0 +1,68 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontStretch'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontStretch
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontStretch
+---
+
+
```

Comment:
Same as above? Extra line of whitespace?

Reply:
they are all made from the same template, so they all have the same mistake... updated it for all! thank you for pointing out.